index.html

```diff
@@ -1168,7 +1168,10 @@ <h2>Class Definitions</h2>
         </p>
 
         <p>
-            Please note that all vocabulary terms and values are case sensitive.
+            <span class="rfc2119-assertion" id="td-processor">
+                Please note that all vocabulary terms and values MUST be case sensitive.
+            </span>
+            
             This is also true for the serialization of the information model (Section <a href="#sec-td-serialization"></a>).
```

Comment:
This looks like another normative requirement. Suggest to either include it in the <span> section above or define another normative <span>

Reply:
This comment seems to be outdated. This is now a normative requirement. Resolving